src/inmanta/ast/statements/define.py

```diff
@@ -367,6 +363,7 @@ def evaluate(self) -> None:
             implement.comment = self.comment
             implement.constraint = self.select
             implement.location = self.entity_location
+            self.anchors.extend(implement.constraint.get_anchors())
```

Comment:
I don't think the statement is normalized yet at this point. It's normalized in `Entity.normalize` (a type).

Damn, this is becoming a bit of a mess now that the definition statements come into it.